src/main/groovy/com/getkeepsafe/dexcount/sdkresolver/SdkResolver.groovy

```diff
@@ -99,26 +115,32 @@ class SdkResolver {
         if (userAndroid.exists()) {
             log.debug "Found existing SDK at '$userAndroid.absolutePath'. Writing to $FN_LOCAL_PROPERTIES."
 
-            writeLocalProperties userAndroid.absolutePath
+            writeLocalProperties(userAndroid.absolutePath, ndkDir)
```

Comment:
I don't think so, ndkDir is not a File like userAndroid. It's just a string picked from environment variable.